### PR TITLE
Gmpe config template

### PR DESCRIPTION
**Pull request by shot0829**
Doc on testing:https://wiki.canterbury.ac.nz/display/QuakeCore/Verification+of+GMPE+parameter+entry+config

### CODE REVIEW

**Review by jasonmotha — APPROVED** [review]
Looks good. Pending testing as per James's review.

**Comment by jasonmotha** [comment]
Also, looks like we need the qcore jenkins fix here too. :sweat:

**Comment by shot0829** [comment]
> Also, looks like we need the qcore jenkins fix here too. sweat

every repo that needs qcore will need update on the jenkin file

File: empirical/util/gmpe_params.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ ASK_14:
     # options : [0 (Global), 1 (California), 2 (Japan), 3 (China), 4 (Italy), 5 (Turkey), 6 (Taiwan)]
     region: 0
     # type : Bool
-    # flag for hanging wall sites. Default=0
+    # flag for hanging wall sites. Null = Default
```
Comment (jpa198) [inline empirical/util/gmpe_params.yaml:28]: Default = Null, but sure

**Review by jpa198 — APPROVED** [review]

**Comment by jpa198** [comment]
I've cleared out the jenkins server again, if you add the qcore fix the tests should work @shot0829

File: empirical/util/gmpe_params.yaml (hunk 1 above)
Comment (jasonmotha) [reply empirical/util/gmpe_params.yaml:28]: I think specifically a longer term plan for Hw sites as it's a site-specific term. But okay as is for now.

**Comment by shot0829** [comment]
https://wiki.canterbury.ac.nz/display/QuakeCore/Verification+of+GMPE+parameter+entry+config

**Pull request merged**